### ISSUE TITLE
Header tiled background image

### ISSUE THREAD

**Issue by designbychris** [akvo/Akvo-Theme#53]
We will sometimes have a tiled image in the background of the header.

Can you set an option to allow this? 

It may be good to have it as a tick box or a dropdown like the Narrow height option :)

**Comment by samvthom16** [comment]
@designbychris Please share a visual reference for the same.

**Comment by designbychris** [comment]
The tiled background is here https://designbychris.co.uk/akvo/newsite/single3.php the header image tiles, its not one large image. If you inspect element on it you will see that its set to repeat, and the image size is only 300px x 300px. The new image will be larger (600px x 600px) but that shouldn't matter if we can set an option to make the background image repeat :)

**Comment by samvthom16** [comment]
@designbychris So the option that needs to be added is if the header image needs to be repeated or to keep it as a cover?

**Comment by designbychris** [comment]
yes indeed :) so if the repeater option is selected, then it will repeat the header image instead of filling it with cover :)

**Comment by samvthom16** [comment]
Continuation of this issue:
https://github.com/akvo/Akvo-Theme/issues/40
